TASK/TaskFollowNavMeshToCoord.md

```diff
@@ -5,21 +5,45 @@ ns: TASK
 
 ```c
 // 0x15D3A79D4E44B913 0xFE4A10D9
-void TASK_FOLLOW_NAV_MESH_TO_COORD(Ped ped, float x, float y, float z, float speed, int timeout, float stoppingRange, BOOL persistFollowing, float unk);
+void TASK_FOLLOW_NAV_MESH_TO_COORD(Ped ped, float x, float y, float z, float MoveBlendRatio, int Time, float Radius, Any Flags, float FinalHeading);
 ```
 
 ```
-If no timeout, set timeout to -1.  
+Sometimes a path may not be able to be found. This could happen because there simply isn't any way to get there, or maybe a bunch of dynamic objects have blocked the way, 
+or maybe the destination is too far away. In this case the ped will simply stand still.
+To identify when this has happened, you can use GET_NAVMESH_ROUTE_RESULT. This will help you find situations where peds cannot get to their target.
```

Comment:
Would be better to have a shorter description. Add the return description on his section too. 

Not sure about the part why path may not be found, any example?

Reply:
i dont have a specific example atm, but from my testing it seems that if you set the end destination to somewhere unaccessable it behaves like this, also seems like R* has checks for this as well in the decompiled scripts

Reply:
What did test? Any repo?

Reply:
it was a couple weeks ago so i think i deleted it by now, but for example, if you have a ped on the ground like a sidewalk, and you try to make it go ontop of a skyscraper it sometimes wont know what to do.